app/javascript/packs/feedadoc.jsx

```diff
@@ -80,6 +81,9 @@ function App() {
                     </>
                   )}
                 />
+                <Route path="/volunteer/confirmation">
+                  <VolunteerConfirmation />
```

Comment:
I think we will probably want the `<ScrollToTopOnMount />` component here to ensure the page loads at the top.